### ISSUE TITLE
getting error with telegrambot

### ISSUE THREAD

**Issue by qwertangel** [windkh/node-red-contrib-telegrambot#62]
Trying to send a message to a group chat, using sender node (already set configuration node).
Also message is not getting sent.
Latest version 5.1.3 of telegram bot is throwing following error on deploy

```
Jan 21 12:33:19 marut node-red[645]: Unhandled rejection TypeError: Cannot read property 'status' of null
Jan 21 12:33:19 marut node-red[645]:     at /home/myhomedir/.node-red/node_modules/node-red-contrib-telegrambot/telegrambot/99-telegrambot.js:292:26
Jan 21 12:33:19 marut node-red[645]:     at Array.forEach (<anonymous>)
Jan 21 12:33:19 marut node-red[645]:     at TelegramBotNode.setUsersStatus (/home/myhomedir/.node-red/node_modules/node-red-contrib-telegrambot/telegrambot/99-telegrambot.js:290:22)
Jan 21 12:33:19 marut node-red[645]:     at /home/myhomedir/.node-red/node_modules/node-red-contrib-telegrambot/telegrambot/99-telegrambot.js:222:34
Jan 21 12:33:19 marut node-red[645]:     at tryCatcher (/home/myhomedir/.node-red/node_modules/bluebird/js/release/util.js:16:23)
Jan 21 12:33:19 marut node-red[645]:     at Promise._settlePromiseFromHandler (/home/myhomedir/.node-red/node_modules/bluebird/js/release/promise.js:512:31)
Jan 21 12:33:19 marut node-red[645]:     at Promise._settlePromise (/home/myhomedir/.node-red/node_modules/bluebird/js/release/promise.js:569:18)
Jan 21 12:33:19 marut node-red[645]:     at Promise._settlePromise0 (/home/myhomedir/.node-red/node_modules/bluebird/js/release/promise.js:614:10)
Jan 21 12:33:19 marut node-red[645]:     at Promise._settlePromises (/home/myhomedir/.node-red/node_modules/bluebird/js/release/promise.js:694:18)
Jan 21 12:33:19 marut node-red[645]:     at _drainQueueStep (/home/myhomedir/.node-red/node_modules/bluebird/js/release/async.js:138:12)
Jan 21 12:33:19 marut node-red[645]:     at _drainQueue (/home/myhomedir/.node-red/node_modules/bluebird/js/release/async.js:131:9)
Jan 21 12:33:19 marut node-red[645]:     at Async._drainQueues (/home/myhomedir/.node-red/node_modules/bluebird/js/release/async.js:147:5)
Jan 21 12:33:19 marut node-red[645]:     at Immediate.Async.drainQueues (/home/myhomedir/.node-red/node_modules/bluebird/js/release/async.js:17:14)
Jan 21 12:33:19 marut node-red[645]:     at runCallback (timers.js:794:20)
Jan 21 12:33:19 marut node-red[645]:     at tryOnImmediate (timers.js:752:5)
Jan 21 12:33:19 marut node-red[645]:     at processImmediate [as _immediateCallback] (timers.js:729:5)
Jan 21 12:33:26 marut node-red[645]: 21 Jan 12:33:26 - [error] [telegram bot:199e8fa8.58486] Error stopping node: TimeoutError: timed out after 15000ms
Jan 21 12:33:26 marut node-red[645]: 21 Jan 12:33:26 - [debug] TimeoutError: timed out after 15000ms
Jan 21 12:33:26 marut node-red[645]:     at onTimeout (/usr/local/lib/node_modules/node-red/node_modules/when/lib/decorators/timed.js:69:7)
Jan 21 12:33:26 marut node-red[645]:     at Timeout._onTimeout (/usr/local/lib/node_modules/node-red/node_modules/when/lib/decorators/timed.js:13:4)
Jan 21 12:33:26 marut node-red[645]:     at ontimeout (timers.js:482:11)
Jan 21 12:33:26 marut node-red[645]:     at tryOnTimeout (timers.js:317:5)
Jan 21 12:33:26 marut node-red[645]:     at Timer.listOnTimeout (timers.js:277:5)
```

Any idea what could be wrong ?
Another problem is after i configure the node, create a sender node, then after that any deploy just hangs forever. Have to kill and restart node-red.

Edit: updated with full stack

**Comment by khaeljy** [comment]
Hi!

I have a similar error since the latest update

```
Unhandled rejection TypeError: Cannot read property 'forEach' of undefined,
    at TelegramBotNode.setUsersStatus (/data/node_modules/node-red-contrib-telegrambot/telegrambot/99-telegrambot.js:290:21),
    at TelegramBot.<anonymous> (/data/node_modules/node-red-contrib-telegrambot/telegrambot/99-telegrambot.js:151:42),
    at TelegramBot.emit (/data/node_modules/eventemitter3/index.js:181:35),
    at TelegramBotPolling._error (/data/node_modules/node-telegram-bot-api/src/telegramPolling.js:92:21),
    at _lastRequest._getUpdates.then.catch.err (/data/node_modules/node-telegram-bot-api/src/telegramPolling.js:121:23),
    at tryCatcher (/data/node_modules/bluebird/js/release/util.js:16:23),
    at Promise._settlePromiseFromHandler (/data/node_modules/bluebird/js/release/promise.js:512:31),
    at Promise._settlePromise (/data/node_modules/bluebird/js/release/promise.js:569:18),
    at Promise._settlePromise0 (/data/node_modules/bluebird/js/release/promise.js:614:10),
    at Promise._settlePromises (/data/node_modules/bluebird/js/release/promise.js:690:18),
    at _drainQueueStep (/data/node_modules/bluebird/js/release/async.js:138:12),
    at _drainQueue (/data/node_modules/bluebird/js/release/async.js:131:9),
    at Async._drainQueues (/data/node_modules/bluebird/js/release/async.js:147:5),
    at Immediate.Async.drainQueues (/data/node_modules/bluebird/js/release/async.js:17:14),
    at runCallback (timers.js:672:20),
    at tryOnImmediate (timers.js:645:5),
    at processImmediate [as _immediateCallback] (timers.js:617:5),
```

**Comment by windkh** [comment]
What nodered version do you use?

**Comment by qwertangel** [comment]
For me 
node version: v8.10.0
Noe-red version : 0.19.5
node-red-contrib-telegrambot: 5.1.3

**Comment by windkh** [comment]
Mine:
node version: v8.11.1
node-red: v0.18.4

Maybe the newer node-red version introduced that problem.... I will update mine to be able to reproduce the problem

**Comment by windkh** [comment]
I released version 5.1.4 that should fix the problem. I tested with nodered v0.18.7 as I could not update to 0.19.x yet. Please could you test this version and provide feedback?

I found out that the problem also occurs when doing a partial deploy instead of a full one.

**Comment by khaeljy** [comment]
It's working for me, thanks!
